libs/community/langchain_community/llms/mlx_pipeline.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, Iterator, List, Mapping, Optional
+
+import mlx.nn as nn
```

Comment:
this should be conditionally imported within a function since it's not a langchain-community dependency

Reply:
Done, I have removed them. I was using them for type annotation and saw that I could use Any instead. ✅

Reply:
Let me know if there any further changes needed :)